gensim/models/doc2vec.py

```diff
@@ -63,25 +67,19 @@
 
 import logging
 import os
-
-try:
-    from queue import Queue
-except ImportError:
-    from Queue import Queue  # noqa:F401
-
 from collections import namedtuple, defaultdict
 from collections.abc import Iterable
 from timeit import default_timer
-from dataclasses import dataclass
 
+from dataclasses import dataclass
 from numpy import zeros, float32 as REAL, vstack, integer, dtype
 import numpy as np
+from six.moves import range
+from six import string_types, integer_types, itervalues
```

Comment:
These `six` imports could easily be dropped with a few other tiny changes, now that only Py3 is supported

Reply:
Right. I though I did it, an omission. I'll push a fix.